src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>NuGet wrapper for dotnet.exe.</Description>
-    <TargetFrameworks>$(TargetFrameworksExe)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksExe);netcoreapp3.0</TargetFrameworks>
```

Comment:
Update the property in the common targets.

Reply:
Thanks! Changed accordingly.